File: 2-Semestre/Taller de Programación/Personal/Python-C#/Ejercicio 2.cs
/*
Conversión de una longitud dada en centímetros a pies.
Los centímetros son ingresados por el usuario. Considere que: 1 pie = 30.48 centímetros.
*/

using System;

namespace Ejercicio_2
{
    public class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Este programa convierte centímetros a pies\n");

            double cm;

            do
            {
                Console.Write("Ingrese la longitud en cm: ");
                cm = Convert.ToDouble(Console.ReadLine());

                if (cm <= 0)
                {
                    Console.WriteLine("[!] La longitud debe ser mayor a 0.");
                }
            } while (cm <= 0);

            double ft = cm / 30.48;

            Console.WriteLine($"La longitud en pies es de {ft} ft");
        }
    }
}

File: 1-Semestre/Desarrollo de algoritmos/Actividades/Diagramas de flujo-Python (Semana 5)/Ejercicio 2.py
'''
a) Conversión de una longitud dada en centímetros a pies.
b) Los centímetros son ingresados por el usuario.
    Considere que: 1 pie = 30.48 centímetros.
'''

print("Este programa convierte centímetros a pies.")
cm = int(input("Ingrese la longitud en cm: "))
ft = 0

while cm <= 0:
    print("[!] La longitud debe ser mayor a 0.")
    cm = int(input("Ingrese la longitud en cm: "))

ft = cm / 30.48
print(f"La longitud en pies es de {ft:.2f} ft")